package.json

```diff
@@ -26,6 +26,8 @@
     "publish:test": "npm test",
     "test": "run-s build build:demo test:jest",
     "test:jest": "jest",
+    "test:jest:update": "jest --updateSnapshot",
+    "test:update": "run-s build build:demo test:jest:update",
```

Comment:
[sand] we might want to use actual objects, this makes it too easy to run this and ignore snapshots and captures implementation details. It might be a huge object but at least the update will be more deliberate